lib/tapioca/helpers/sorbet_helper.rb

```diff
@@ -24,7 +24,6 @@ module SorbetHelper
     FEATURE_REQUIREMENTS = T.let(
       {
         # feature_name: ::Gem::Requirement.new(">= ___"), # https://github.com/sorbet/sorbet/pull/___
-        non_generic_weak_map: ::Gem::Requirement.new(">= 0.5.10587"), # https://github.com/sorbet/sorbet/pull/6610
```

Comment:
If we are bumping to `>= 0.5.10566`, we can't remove this feature check since it requires `>= 0.5.10587`.